src/index.html

```diff
@@ -4,9 +4,410 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a class="header__logo" href="#">
+          <img src="/src/images/logo.png" alt="moyo">
+        </a>
+        <nav class="header__nav nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link is-active" href="#apple">Apple</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link"  href="#samsung">Samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#smartphones">Smartphones</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#laptops&computers"
+                data-qa="nav-hover">Laptops & Computers
+              </a>
```

Comment:
```suggestion
              <a
                class="nav__link"
                href="#laptops&computers"
                data-qa="nav-hover"
              >
                Laptops & Computers
              </a>
```
Fix formatting in all places